determined-ai/determined
# feat: Add selection label to FlatRuns page

## Ticket: [ET-309](https://hpe-aiatscale.atlassian.net/browse/ET-309)




## Description


Add the selection label to the flat runs table.


## Test Plan

- Enable the Flat runs feature
- go to a runs table
  - there should be a label with the total number of runs
- make a selection
  - the label should be updated accordingly 



![Screenshot 2024-06-20 at 15 48 22](https://github.com/determined-ai/determined/assets/104855841/c85537b9-406e-4d5f-8e2f-1ea09c6cba95)

![Screenshot 2024-06-20 at 15 48 12](https://github.com/determined-ai/determined/assets/104855841/5840a6a7-7b9e-4ed5-b996-2cd66e8fb001)


## Checklist

  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.




[DET-123]: https://hpe-aiatscale.atlassian.net/browse/DET-123?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[ET-252]: https://hpe-aiatscale.atlassian.net/browse/ET-252?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[ET-309]: https://hpe-aiatscale.atlassian.net/browse/ET-309?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
